2019-07-13 13:01:02.070 INFO _internal - _log:  * Running on http://0.0.0.0:8031/ (Press CTRL+C to quit)
2019-07-13 13:26:22.341 ERROR app - log_exception: Exception on / [GET]
Traceback (most recent call last):
  File "/home/ykiveish/.local/lib/python2.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ykiveish/.local/lib/python2.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ykiveish/.local/lib/python2.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ykiveish/.local/lib/python2.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ykiveish/.local/lib/python2.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ykiveish/mks/nodes/101/mksdk/MkSAbstractNode.py", line 23, in __call__
    return render_template(self.Page, data=self.DataToJS)
  File "/home/ykiveish/.local/lib/python2.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/ykiveish/.local/lib/python2.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/ykiveish/.local/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/ykiveish/.local/lib/python2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/ykiveish/.local/lib/python2.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/ykiveish/.local/lib/python2.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/home/ykiveish/.local/lib/python2.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
TemplateNotFound: index.html
2019-07-13 13:26:22.350 INFO _internal - _log: 192.168.0.102 - - [13/Jul/2019 13:26:22] "GET / HTTP/1.1" 500 -
2019-07-13 13:26:22.419 INFO _internal - _log: 192.168.0.102 - - [13/Jul/2019 13:26:22] "GET /favicon.ico HTTP/1.1" 404 -
2019-07-13 13:26:57.528 INFO _internal - _log: 192.168.0.102 - - [13/Jul/2019 13:26:57] "GET /get/node_info/ HTTP/1.1" 404 -
2019-07-13 13:28:29.184 INFO _internal - _log: 192.168.0.102 - - [13/Jul/2019 13:28:29] "GET /get/node_info/ac6de837-7863-72a9-c789-a0aae7e9d93e HTTP/1.1" 200 -
2019-07-13 13:28:45.862 INFO _internal - _log: 192.168.0.102 - - [13/Jul/2019 13:28:45] "GET /get/node_info/ac6de837-7863-72a9-c789-a0aae7e9d93e HTTP/1.1" 200 -
2019-07-13 13:28:48.920 INFO _internal - _log: 192.168.0.102 - - [13/Jul/2019 13:28:48] "GET /get/node_info/ac6de837-7863-72a9-c789-a0aae7e9d93e HTTP/1.1" 200 -
